internal/testutils/path.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// MinimalPathEnv is the minimal PATH environment variable used to run tests.
+const MinimalPathEnv = "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
We're appending to this also the `$(go env GOPATH)/bin`, right? Since that's where most of my test binaries are... It seems so though.

EDIT: I see we do use that